pkg/serializer/serializer.go

```diff
@@ -171,6 +171,10 @@ func NewSerializer(forwarder forwarder.Forwarder, orchestratorForwarder, contlcy
 		log.Warn("JSON to V1 intake is disabled: all payloads to that endpoint will be dropped")
 	}
 
+	if !config.Datadog.GetBool("enable_sketch_stream_payload_serialization") {
+		log.Warn("'enable_sketch_stream_payload_serialization' is set to false which is not recommended. This option will be deprecated and removed in the future. If you need this option, please reach out to support or file an issue on github.com/DataDog/datadog-agent")
```

Comment:
If we logging a warning, I think it's safe to say the option is deprecated already. It may also make sense to mention this in the release notes.

```suggestion
		log.Warn("'enable_sketch_stream_payload_serialization' is set to false which is not recommended. This option is deprecated and will removed in the future. If you need this option, please reach out to support")
```